.circleci/config.yml

```diff
@@ -68,8 +72,77 @@ jobs:
           no_output_timeout: 20m
       - run: make ci-build-and-upload-cli
 
+  e2e-tests-aws:
+    docker:
+      - image: circleci/python:3.6
+    environment:
+      CORTEX_TEST_REALTIME_DEPLOY_TIMEOUT: 60
+      CORTEX_TEST_BATCH_DEPLOY_TIMEOUT: 30
+      CORTEX_TEST_BATCH_JOB_TIMEOUT: 200
+      NUM_BUILD_PROCS: 2
+    steps:
+      - checkout
+      - setup_remote_docker:
+          docker_layer_caching: true
+      - install-go
+      - run:
+          name: Generate Cluster Config
+          command: |
+            cat \<< EOF > ./cluster-aws.yaml
+            cluster_name: cortex
+            provider: aws
+            region: ${AWS_REGION}
```

Comment:
I don't mind hardcoding the region and the bucket, or if you prefer to use the environment variables, maybe they should be more clearly just for this purpose, e.g. `NIGHTLY_AWS_REGION`, what do you think?

Reply:
that makes sense